features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_renamed.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous branch is renamed during a Git Town command, the main branch becomes the new previous branch
```

Comment:
It makes `main` the previous branch because there's no information on what it *should* be, so it defaults to something sane. And actually, `git-rename-branches` checks out the branch already, so perhaps you'd want to use the `n-2` branch in history and that's a path I don't want to go down.

If you do this in vanilla git, you'll see that you get an error checking out a renamed branch. So instead of that, we just default to `main`.

```bash
foo$ git checkout bar
bar$ git branch foo -m foo-renamed
bar$ git checkout -
error: pathspec '-' did not match any file(s) known to git.
```